### DISCUSSION TITLE
Not show dev tools on each run?

### DISCUSSION THREAD

**Discussion by skythomp16** [nklayman/vue-cli-plugin-electron-builder#1400] [Q&A] [answered]
Hey guys,

Loving this plugin but is there any way to NOT open the dev tools every time the project is ran with electron:serve?  I tried looking at the dist_electron/index.js code and couldn't see where that call was happening but there is a LOT of code in that file.

Skyler

**Comment by nklayman** [comment, marked as answer]
Remove this line from your `src/background.js` file:
```js
if (!process.env.IS_TEST) win.webContents.openDevTools()
```

The `dist_electron/index.js` is generated by webpack using the `src/background.js` file as the entry.
